src/faebryk/libs/kicad/fileformats.py

```diff
@@ -660,6 +660,7 @@ class C_gr_poly(C_polygon):
         options: Optional[C_options] = None
         primitives: Optional[C_gr] = None
         # TODO: primitives: add: gr_line, gr_arc, gr_circle, gr_rect, gr_curve, gr_bbox
+        unknown: CatchAll = None
```

Comment:
We're abusing this catch-all type from `dataclass_json`. Can make our own if we care. It's just a type (or really a pair)